subprojects/android-test/test-app/build.gradle.kts

```diff
@@ -46,7 +46,7 @@ android {
         }
     }
 
-    testBuildType = "staging"
+    testBuildType = getOptionalStringProperty("avito.testBuildType", "staging")
```

Comment:
Why "avito"? This module and logic knows nothing about avito
What about simple "testBuildType"?